package.json

```diff
@@ -241,7 +241,8 @@
         "intl": "^1.2.5",
         "intl-messageformat": "^7.8.4",
         "intl-messageformat-parser": "^3.6.4",
-        "jest": "^25.1.0",
+        "jest": "29.5.0",
```

Comment:
is there a reason why we're pinning the version rather than accept minor and patch updates?

Reply:
its dependencies like `@jest/core` are installs the latest versions - `29.7.0`, why not change this to a caret: `"jest": "^29.5.0"` so they're kept in sync?